### DISCUSSION TITLE
About changing repo name

### DISCUSSION THREAD

**Discussion by jimmyyzr** [community/community#23177] [New to GitHub]
<p>When I was working on my lab’s server, I was always asked for my username and password, which is annoying. After I followed some StackOverflow instructions, I seemed to change my repo name and wasn’t able to push and pull properly. How should I fix this? Thanks in advance.<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/3/b/3bbf4faee5bff3d459cd9ea34f5ccd53b1650d96.png" data-download-href="/uploads/short-url/8wy3iG0zHLtHxv2PDomQ2erxQXk.png?dl=1" title="Screen Shot 2020-11-19 at 10.39.31 AM" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/43551018/181075459-2e50a35d-f576-4c4b-b533-784845fd4c69.png" alt="Screen Shot 2020-11-19 at 10.39.31 AM" data-base62-sha1="8wy3iG0zHLtHxv2PDomQ2erxQXk" width="690" height="103" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/3/b/3bbf4faee5bff3d459cd9ea34f5ccd53b1650d96_2_690x103.png, https://aws1.discourse-cdn.com/github/optimized/3X/3/b/3bbf4faee5bff3d459cd9ea34f5ccd53b1650d96_2_1035x154.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/3/b/3bbf4faee5bff3d459cd9ea34f5ccd53b1650d96_2_1380x206.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/3/b/3bbf4faee5bff3d459cd9ea34f5ccd53b1650d96_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2020-11-19 at 10.39.31 AM</span><span class="informations">2164×326 26.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by airtower-luna** [comment, marked as answer]
<p>It’s hard to say for sure without knowing exactly what you tried, but it looks like you changed the remote repository URL and made a mistake while doing so, leading to an invalid URL. You can obtain the correct URLs for HTTPS or SSH by going to your repository page on Github. Click on the green “Code” button and choose the protocol you want in the popup.</p>
<p>To learn how to configure the remote URL, check:<br>
</p><aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://git-scm.com/favicon.ico" class="site-icon" width="16" height="16">
      <a href="https://git-scm.com/book/en/v2/Git-Basics-Working-with-Remotes" target="_blank" rel="noopener">git-scm.com</a>
  </header>
  <article class="onebox-body">
    <img src="" class="thumbnail" width="16" height="16">

<h3><a href="https://git-scm.com/book/en/v2/Git-Basics-Working-with-Remotes" target="_blank" rel="noopener">Git - Working with Remotes</a></h3>



  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by jimmyyzr** [comment]
<p>Thanks. I have read the link you attached. I wonder if I need to use <code>git remote add</code>? I am really worried to mess up my current project.</p>

**Comment by airtower-luna** [comment]
<p>The problem you described looks more like you need to fix the URL, but as I said it’s hard to say without more detail. A good start would be to look at the output of <code>git remote -v</code>. That shows the push and fetch URLs for all remotes you have configured, which should reveal any obvious errors.</p>
